pkg/agent/baker.go

```diff
@@ -826,6 +826,19 @@ func getContainerServiceFuncMap(config *datamodel.NodeBootstrappingConfiguration
 			}
 			return 0
 		},
+		"ShouldConfigureCustomCATrust": func() bool {
+			return config.CustomCATrustConfig != nil && len(config.CustomCATrustConfig.CustomCATrustCerts) > 0
```

Comment:
nit: consistency on the nil/empty check on CustomCATrustCerts. apparently len(<nil slice>) will return 0,  so len(<slice>) == 0 and <slice> == nil actually are the same.

Reply:
could also refactor that check out to its own function and reuse within "GetCustomCATrustConfigCerts"